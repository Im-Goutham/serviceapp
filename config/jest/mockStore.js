export const initialState = {
    user: {
      user: {},
      backButton: false
    },
    photo: {
      photoView: false,
      images: [],
      index: 0
    }
  }