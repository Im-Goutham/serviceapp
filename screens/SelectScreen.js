import React, { Component } from 'react';
import {
    View,
    StyleSheet,
    ScrollView,
    Image,
    TouchableOpacity,
    Dimensions,
    Platform,
    Text, TouchableHighlight
} from 'react-native';
import JobsList from '../components/JobsList';
import Header from '../components/Header';
import LinearGradient from 'react-native-linear-gradient';
import { connect } from 'react-redux';
import * as actions from '../actions';
import SearchBar from '../components/SearchBar';
import HeaderScreen from './HeaderScreen';
import { scale } from '../global';
let back_arrow = require('../assets/icons/arrow_left.png');
let menu = require('../assets/icons/menu.png');

class SelectScreen extends Component {
    constructor(props) {
        super(props);
        this.state = {
            tabindex: 0
        }
    }
    navigateToScreen = (route) => () => {
        // const navigateAction = NavigationActions.navigate({
        //   routeName: route
        // });
        // console.log('route is ',route);
        // const navigateAction = NavigationActions.navigate({
        //   routeName: route,
        //   params: {backButton: false},
        //   action: NavigationActions.navigate({ routeName: route, params: {backButton: false}})
        // })
        // // this.props.navigation.navigate(route,{backButton: true});
        // this.props.navigation.dispatch(navigateAction);
        if (route == 'account') {
            this.props.showBackButton(true);
        }
        this.props.navigation.navigate(route);
    }
    render() {
        let { backButton } = this.props;
        return (
            <View style={{ flex: 1 }}>
                <LinearGradient
                    colors={['#3E85EF', '#3EBDEF']}
                    start={{ x: 0, y: 0 }}
                    end={{ x: 1, y: 0 }}
                    style={{
                        flex: 1
                    }}>
                    <HeaderScreen
                        header={
                            <Header
                                navigation={this.props.navigation}
                                left={
                                    <View style={{ backgroundColor: 'transparent', justifyContent: "center", alignItems: 'center', flexDirection: "row" }}>
                                        <TouchableOpacity onPress={() => this.props.navigation.goBack()} style={{ width: "50%", height: scale(54), backgroundColor: 'transparent', justifyContent: "center", alignItems: 'center' }}>
                                            <Image source={back_arrow} style={{ width: '50%', height: scale(20) }} resizeMode="contain" resizeMethod="resize" />
                                        </TouchableOpacity>
                                        <TouchableOpacity onPress={() => this.props.navigation.openDrawer()} style={{ width: !backButton ? scale(54) : "50%", height: scale(54), backgroundColor: 'transparent', justifyContent: "center", alignItems: 'center' }}>
                                            <Image source={menu} style={{ width: !backButton ? '100%' : '50%', height: scale(20) }} resizeMode="contain" resizeMethod="resize" />
                                        </TouchableOpacity>
                                    </View>
                                }
                                title={
                                    <View style={{ justifyContent: 'center', alignItems: 'flex-start', height: scale(54) }}>
                                        <Text style={{ fontFamily: 'Montserrat-Bold', color: "#fff", fontSize: scale(20) }}>Select a Job</Text>
                                    </View>
                                }
                                right={
                                    null
                                }
                            />
                        }
                        content={
                            <View style={{ backgroundColor: "transparent", justifyContent: "space-between", paddingTop: scale(10), paddingBottom: scale(20), marginHorizontal: scale(10) }}>
                                <SearchBar placeholder={'Search from your jobs...'} />
                            </View>
                        }
                    />
                    <View style={{ backgroundColor: "#009933", flex: 1 }}>
                        <JobsList type='select' navigation={this.props.navigation} />
                    </View>
                </LinearGradient>
            </View>
        )
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1
    },
    modal: {
        // justifyContent: 'center',
        // alignItems: 'center'
    },

    modal2: {
        height: scale(230),
        backgroundColor: "#3B5998"
    },

    modal3: {
        height: scale(300),
        width: scale(300)
    },
    button: {
        justifyContent: 'center',
        alignItems: 'center',
        width: scale(100),
        height: scale(40),
        borderRadius: scale(20),
        // borderWidth: 1,
        // borderColor: '#008000',
        paddingTop: scale(5),
        paddingBottom: scale(5),
    },
})
const mapStateToProps = state => ({
    backButton: state.user.backButton,
})
export default connect(mapStateToProps, actions)(SelectScreen);
