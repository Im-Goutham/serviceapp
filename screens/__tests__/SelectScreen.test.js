jest.unmock('redux-mock-store')
jest.unmock('redux-thunk')
import React from 'react';
import { shallow, mount, render } from 'enzyme';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import renderer from 'react-test-renderer';
import SelectScreen from '../SelectScreen';
import {initialState} from '../../config/jest/mockStore';

let store;
const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

describe('SelectScreen', () => {
    let wrapper;
    beforeEach(() => {
        store = mockStore(initialState);
        wrapper = shallow(
            <SelectScreen
                store={store}
                // navigation={{state: {params: {mainScreen:''}}}}
                />).dive();
    //    wrapper.setProps({navigation:});
    })
    it('snapshot',()=> {
        expect(wrapper).toMatchSnapshot();
    })
});
