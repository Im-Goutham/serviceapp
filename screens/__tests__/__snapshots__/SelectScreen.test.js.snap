// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectScreen snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SelectScreen
    backButton={false}
    hidePhotoView={[Function]}
    showBackButton={[Function]}
    showPhotoView={[Function]}
    signIn={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    storeSubscription={
      Subscription {
        "listeners": Object {
          "clear": [Function],
          "get": [Function],
          "notify": [Function],
          "subscribe": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": [Function],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <LinearGradient
        colors={
          Array [
            "#3E85EF",
            "#3EBDEF",
          ]
        }
        end={
          Object {
            "x": 1,
            "y": 0,
          }
        }
        start={
          Object {
            "x": 0,
            "y": 0,
          }
        }
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <HeaderScreen
          content={
            <Component
              style={
                Object {
                  "backgroundColor": "transparent",
                  "justifyContent": "space-between",
                  "marginHorizontal": 8.695652173913045,
                  "paddingBottom": 17.39130434782609,
                  "paddingTop": 8.695652173913045,
                }
              }
            >
              <SearchBar
                placeholder="Search from your jobs..."
              />
            </Component>
          }
          header={
            <Header
              left={
                <Component
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "flexDirection": "row",
                      "justifyContent": "center",
                    }
                  }
                >
                  <TouchableOpacity
                    activeOpacity={0.2}
                    onPress={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                        "width": "50%",
                      }
                    }
                  >
                    <Image
                      resizeMethod="resize"
                      resizeMode="contain"
                      source={
                        Object {
                          "testUri": "../../../assets/icons/arrow_left.png",
                        }
                      }
                      style={
                        Object {
                          "height": 17.39130434782609,
                          "width": "50%",
                        }
                      }
                    />
                  </TouchableOpacity>
                  <TouchableOpacity
                    activeOpacity={0.2}
                    onPress={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                        "width": 46.95652173913044,
                      }
                    }
                  >
                    <Image
                      resizeMethod="resize"
                      resizeMode="contain"
                      source={
                        Object {
                          "testUri": "../../../assets/icons/menu.png",
                        }
                      }
                      style={
                        Object {
                          "height": 17.39130434782609,
                          "width": "100%",
                        }
                      }
                    />
                  </TouchableOpacity>
                </Component>
              }
              right={null}
              title={
                <Component
                  style={
                    Object {
                      "alignItems": "flex-start",
                      "height": 46.95652173913044,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Component
                    style={
                      Object {
                        "color": "#fff",
                        "fontFamily": "Montserrat-Bold",
                        "fontSize": 17.39130434782609,
                      }
                    }
                  >
                    Select a Job
                  </Component>
                </Component>
              }
            />
          }
        />
        <Component
          style={
            Object {
              "backgroundColor": "#009933",
              "flex": 1,
            }
          }
        >
          <JobsList />
        </Component>
      </LinearGradient>,
      "style": Object {
        "flex": 1,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <HeaderScreen
            content={
              <Component
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "justifyContent": "space-between",
                    "marginHorizontal": 8.695652173913045,
                    "paddingBottom": 17.39130434782609,
                    "paddingTop": 8.695652173913045,
                  }
                }
              >
                <SearchBar
                  placeholder="Search from your jobs..."
                />
              </Component>
            }
            header={
              <Header
                left={
                  <Component
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "flexDirection": "row",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": "50%",
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/arrow_left.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "50%",
                          }
                        }
                      />
                    </TouchableOpacity>
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": 46.95652173913044,
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/menu.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "100%",
                          }
                        }
                      />
                    </TouchableOpacity>
                  </Component>
                }
                right={null}
                title={
                  <Component
                    style={
                      Object {
                        "alignItems": "flex-start",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Component
                      style={
                        Object {
                          "color": "#fff",
                          "fontFamily": "Montserrat-Bold",
                          "fontSize": 17.39130434782609,
                        }
                      }
                    >
                      Select a Job
                    </Component>
                  </Component>
                }
              />
            }
          />,
          <Component
            style={
              Object {
                "backgroundColor": "#009933",
                "flex": 1,
              }
            }
          >
            <JobsList />
          </Component>,
        ],
        "colors": Array [
          "#3E85EF",
          "#3EBDEF",
        ],
        "end": Object {
          "x": 1,
          "y": 0,
        },
        "start": Object {
          "x": 0,
          "y": 0,
        },
        "style": Object {
          "flex": 1,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "content": <Component
              style={
                Object {
                  "backgroundColor": "transparent",
                  "justifyContent": "space-between",
                  "marginHorizontal": 8.695652173913045,
                  "paddingBottom": 17.39130434782609,
                  "paddingTop": 8.695652173913045,
                }
              }
            >
              <SearchBar
                placeholder="Search from your jobs..."
              />
            </Component>,
            "header": <Header
              left={
                <Component
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "flexDirection": "row",
                      "justifyContent": "center",
                    }
                  }
                >
                  <TouchableOpacity
                    activeOpacity={0.2}
                    onPress={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                        "width": "50%",
                      }
                    }
                  >
                    <Image
                      resizeMethod="resize"
                      resizeMode="contain"
                      source={
                        Object {
                          "testUri": "../../../assets/icons/arrow_left.png",
                        }
                      }
                      style={
                        Object {
                          "height": 17.39130434782609,
                          "width": "50%",
                        }
                      }
                    />
                  </TouchableOpacity>
                  <TouchableOpacity
                    activeOpacity={0.2}
                    onPress={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                        "width": 46.95652173913044,
                      }
                    }
                  >
                    <Image
                      resizeMethod="resize"
                      resizeMode="contain"
                      source={
                        Object {
                          "testUri": "../../../assets/icons/menu.png",
                        }
                      }
                      style={
                        Object {
                          "height": 17.39130434782609,
                          "width": "100%",
                        }
                      }
                    />
                  </TouchableOpacity>
                </Component>
              }
              right={null}
              title={
                <Component
                  style={
                    Object {
                      "alignItems": "flex-start",
                      "height": 46.95652173913044,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Component
                    style={
                      Object {
                        "color": "#fff",
                        "fontFamily": "Montserrat-Bold",
                        "fontSize": 17.39130434782609,
                      }
                    }
                  >
                    Select a Job
                  </Component>
                </Component>
              }
            />,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <JobsList />,
            "style": Object {
              "backgroundColor": "#009933",
              "flex": 1,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "navigation": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <LinearGradient
          colors={
            Array [
              "#3E85EF",
              "#3EBDEF",
            ]
          }
          end={
            Object {
              "x": 1,
              "y": 0,
            }
          }
          start={
            Object {
              "x": 0,
              "y": 0,
            }
          }
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <HeaderScreen
            content={
              <Component
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "justifyContent": "space-between",
                    "marginHorizontal": 8.695652173913045,
                    "paddingBottom": 17.39130434782609,
                    "paddingTop": 8.695652173913045,
                  }
                }
              >
                <SearchBar
                  placeholder="Search from your jobs..."
                />
              </Component>
            }
            header={
              <Header
                left={
                  <Component
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "flexDirection": "row",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": "50%",
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/arrow_left.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "50%",
                          }
                        }
                      />
                    </TouchableOpacity>
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": 46.95652173913044,
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/menu.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "100%",
                          }
                        }
                      />
                    </TouchableOpacity>
                  </Component>
                }
                right={null}
                title={
                  <Component
                    style={
                      Object {
                        "alignItems": "flex-start",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Component
                      style={
                        Object {
                          "color": "#fff",
                          "fontFamily": "Montserrat-Bold",
                          "fontSize": 17.39130434782609,
                        }
                      }
                    >
                      Select a Job
                    </Component>
                  </Component>
                }
              />
            }
          />
          <Component
            style={
              Object {
                "backgroundColor": "#009933",
                "flex": 1,
              }
            }
          >
            <JobsList />
          </Component>
        </LinearGradient>,
        "style": Object {
          "flex": 1,
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <HeaderScreen
              content={
                <Component
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "justifyContent": "space-between",
                      "marginHorizontal": 8.695652173913045,
                      "paddingBottom": 17.39130434782609,
                      "paddingTop": 8.695652173913045,
                    }
                  }
                >
                  <SearchBar
                    placeholder="Search from your jobs..."
                  />
                </Component>
              }
              header={
                <Header
                  left={
                    <Component
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "flexDirection": "row",
                          "justifyContent": "center",
                        }
                      }
                    >
                      <TouchableOpacity
                        activeOpacity={0.2}
                        onPress={[Function]}
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "transparent",
                            "height": 46.95652173913044,
                            "justifyContent": "center",
                            "width": "50%",
                          }
                        }
                      >
                        <Image
                          resizeMethod="resize"
                          resizeMode="contain"
                          source={
                            Object {
                              "testUri": "../../../assets/icons/arrow_left.png",
                            }
                          }
                          style={
                            Object {
                              "height": 17.39130434782609,
                              "width": "50%",
                            }
                          }
                        />
                      </TouchableOpacity>
                      <TouchableOpacity
                        activeOpacity={0.2}
                        onPress={[Function]}
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "transparent",
                            "height": 46.95652173913044,
                            "justifyContent": "center",
                            "width": 46.95652173913044,
                          }
                        }
                      >
                        <Image
                          resizeMethod="resize"
                          resizeMode="contain"
                          source={
                            Object {
                              "testUri": "../../../assets/icons/menu.png",
                            }
                          }
                          style={
                            Object {
                              "height": 17.39130434782609,
                              "width": "100%",
                            }
                          }
                        />
                      </TouchableOpacity>
                    </Component>
                  }
                  right={null}
                  title={
                    <Component
                      style={
                        Object {
                          "alignItems": "flex-start",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Component
                        style={
                          Object {
                            "color": "#fff",
                            "fontFamily": "Montserrat-Bold",
                            "fontSize": 17.39130434782609,
                          }
                        }
                      >
                        Select a Job
                      </Component>
                    </Component>
                  }
                />
              }
            />,
            <Component
              style={
                Object {
                  "backgroundColor": "#009933",
                  "flex": 1,
                }
              }
            >
              <JobsList />
            </Component>,
          ],
          "colors": Array [
            "#3E85EF",
            "#3EBDEF",
          ],
          "end": Object {
            "x": 1,
            "y": 0,
          },
          "start": Object {
            "x": 0,
            "y": 0,
          },
          "style": Object {
            "flex": 1,
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "content": <Component
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "justifyContent": "space-between",
                    "marginHorizontal": 8.695652173913045,
                    "paddingBottom": 17.39130434782609,
                    "paddingTop": 8.695652173913045,
                  }
                }
              >
                <SearchBar
                  placeholder="Search from your jobs..."
                />
              </Component>,
              "header": <Header
                left={
                  <Component
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "flexDirection": "row",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": "50%",
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/arrow_left.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "50%",
                          }
                        }
                      />
                    </TouchableOpacity>
                    <TouchableOpacity
                      activeOpacity={0.2}
                      onPress={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "height": 46.95652173913044,
                          "justifyContent": "center",
                          "width": 46.95652173913044,
                        }
                      }
                    >
                      <Image
                        resizeMethod="resize"
                        resizeMode="contain"
                        source={
                          Object {
                            "testUri": "../../../assets/icons/menu.png",
                          }
                        }
                        style={
                          Object {
                            "height": 17.39130434782609,
                            "width": "100%",
                          }
                        }
                      />
                    </TouchableOpacity>
                  </Component>
                }
                right={null}
                title={
                  <Component
                    style={
                      Object {
                        "alignItems": "flex-start",
                        "height": 46.95652173913044,
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Component
                      style={
                        Object {
                          "color": "#fff",
                          "fontFamily": "Montserrat-Bold",
                          "fontSize": 17.39130434782609,
                        }
                      }
                    >
                      Select a Job
                    </Component>
                  </Component>
                }
              />,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <JobsList />,
              "style": Object {
                "backgroundColor": "#009933",
                "flex": 1,
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "navigation": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
